AdobeStockAsset/Test/Mftf/Test/UserConfigureAdobeStockIntegration.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="UserConfigureAdobeStockIntegration">
+        <annotations>
+            <features value="AdobeStockImageConfiguration"/>
+            <stories value="Configure Adobe Stock Integration via the Admin"/>
+            <title value="Admin should be able to configure Adobe Stock Integration"/>
+            <description value="Admin should be able to configure Adobe Stock Integration"/>
+            <severity value="CRITICAL"/>
+            <group value="AdobeStockIntegration"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginActionGroup" stepKey="loginGetFromGeneralFile"/>
+        </before>
+        <after>
+            <actionGroup ref="logout" stepKey="logout"/>
+        </after>
+        <amOnPage url="{{AdminConfigSystemPage.url}}" stepKey="navigateToSystemConfigurationPage" />
+        <waitForPageLoad stepKey="waitForPageLoad"/>
+        <scrollTo selector="{{AdminConfigSystemSection.adobeStockIntegration}}" stepKey="scrollToAdobeStockIntegrationSection"/>
+        <conditionalClick stepKey="expandadobeStockIntegrationTab" selector="{{AdminConfigSystemSection.adobeStockIntegration}}" dependentSelector="{{AdminConfigSystemSection.CheckIfTabExpand}}" visible="true" />
+        <seeElement selector="{{AdminConfigSystemSection.expandedTab}}" stepKey="seeSectionExpanded"/>
+        <fillField userInput="" selector="{{AdminConfigSystemSection.adobeStockApiKey}}" stepKey="fillblankadobeStockApiKey"/>
```

Comment:
It's better to move assertions and actions to action groups.

> Build tests from action groups
> Build your tests using action groups, even if an action group contains a single action.
> 
> Why? For extension developers, this will make it easier to extend or customize tests. Extending a single action group will update all tests that use this group. This improves maintainability as multiple instances of a failure can be fixed with a single action group update.

Reply:
Hello @engcom-Golf ,

Thanks for the feedback. I have created action groups as per the comment. Please let me know if I've made any mistake

Thanks